posthog/hogql/database/schema/persons_pdi.py

```diff
@@ -41,10 +42,21 @@ def persons_pdi_join(
     if not requested_fields:
         raise ResolutionError("No fields requested from person_distinct_ids")
     join_expr = ast.JoinExpr(table=persons_pdi_select(requested_fields))
-    if context.modifiers.personsJoinMode == PersonsJoinMode.left:
-        join_expr.join_type = "LEFT JOIN"
-    else:
+    organization: Organization = context.team.organization if context.team else None
+    # TODO: Remove flag check and use left join for all once deletes are caught up
+    use_inner_join = (
+        posthoganalytics.feature_enabled(
+            "personless-events-not-supported",
+            str(context.team.uuid),
```

Comment:
We don't have the user requesting the query in the context. Because this is a temporary thing, I figure we can just use the team UUID as the distinct ID instead of passing the user ID all the way down. This will result in a new person being made for every team that hits this code when the feature flag is checked. I think that's okay.... but lmk if not.

Reply:
Make this a group flag that works by org / team_id, only evaluate locally, and don't send flag events - this will make it faster, more reliable, and not send events thus not creating new persons.

For an example: https://github.com/PostHog/posthog/blob/master/posthog/caching/utils.py#L74

Reply:
Oh good eye, I just changed to just use `organization.id`. Thanks!